### ISSUE TITLE
Possible conflict with customizer -> site icon

### ISSUE THREAD

**Issue by scofennell** [adamsilverstein/modern-images-wp#27]
Hi, see screenshot where I'm getting a console error when I provide a png favicon in the customizer.

`Uncaught TypeError: Cannot read properties of undefined (reading 'site_icon-32')`

I'm not sure what to make of this, but it seems like a bit of a roadblock.  If I remove the plugin, I can upload and pick my favicon just fine.  And of course, if I config the plugin to no longer convert png's, it works fine as well.

If the `image_editor_output_format` exposed more information about the image and the current screen, I might be able to conditional my way out of this.  As it is now, I'm not sure what a good workaround would be.

<img width="2048" alt="Screen Shot 2021-11-22 at 3 36 39 PM" src="https://user-images.githubusercontent.com/1585637/142931689-1b83eb4c-7af2-4db6-9b83-21f661539287.png">

**Comment by adamsilverstein** [comment]
@scofennell thanks for the bug report. do you have the plugin set to change PNG to WebP (in settings)? Can you provide the PNG image you are using in your case and I will try to reproduce.

**Comment by scofennell** [comment]
Yes, I do have it set to change png to webp.  I was able to repro with any arbitrary png.  Here's one:


![favicon-1](https://user-images.githubusercontent.com/1585637/144093183-a54cb323-3f4c-483a-87b8-4d10c3888fd4.png)
